### ISSUE TITLE
Failed to Created Node Environment

### ISSUE THREAD

**Issue by dgriff20** [elastic/elasticsearch#14533]
When trying to run Elasticsearch for the first time I am receiving the following error...

"Exception in thread "main" java.lang.IllegalStateException: Failed to created node environment
Likely root cause: jave.nio.file.AccessDeniedException: /etc/elasticsearch-2.0.0/data/elasticsearch/nodes"

In looking into the Elasticsearch folders, "nodes" doesn't even exist. Did this somehow unpack incorrectly?

**Comment by rmuir** [comment]
unix file permissions are incorrect in this case.

**Comment by dgriff20** [comment]
rmuir - Thank you for your response. I'm a bit confused on how that could happen. I downloaded everything under my profile and believe I am the owner. At any rate, what would be a good way to fix it?

**Comment by dgriff20** [comment]
Additional errors in here read...

log4j:WARN No appenders could be found for logger (bootstrap)
log4j:WARN Please initialize the log4j system properly.

Then it goes on to be unable to access a folder that does not exist yet.

Unable to access 'path.logs' (usr/share/elasticsearch.logs)

Interestingly, if I make that directory, it moves on and stops at the NEXT directory it can't make/find.

**Comment by clintongormley** [comment]
@dgriff20 it sounds like you're using a package to install Elasticsearch, but then starting it with bin/elasticsearch directly, instead of using the system start/stop scripts.

**Comment by peasead** [comment]
@dgriff20 This doesn't appear to have been resolved with an update from you, but in the event that anyone else is having this issue:

I ran across this while I was learning about ELK and doing a few different variations of the installation. When I ran `yum remove elasticsearch` it left a lot of config files and directories around. 

To fix this issue, before starting for the first time (after a `yum remove elasticsearch`), I manually removed `/var/lib/elasticsearch`, `/etc/elasticsearch`, and `/usr/share/elasticsearch`. After that I proceeded with my fresh installation.

I **completely believe** that there are better ways to do all of this; but I was just learning and when removing and reinstalling ran into this problem...above is what fixed it for me. 

Hope that helps.
